puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/releaseTarballs.groovy

```diff
@@ -0,0 +1,61 @@
+pipeline {
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        parallel {
```

Comment:
Can parallel be inside a `stages` ? The only examples I ever see have it inside a `stage`

Reply:
i thought i saw that somewhere ... but now i can't find it.

Reply:
okay, yeah i can't find it. i'll switch it up.